index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+  <style>
+      *{
+    margin-left: 30px;
+    padding: 0;
+}
+
+body{
+    background-color:#eee ;
+
+}
+ul{
+  text-decoration: none;
+}
+
+#nav-bar{
+    top: 0px;
+    display: flex;
+    width: 100%;
+}
+
+@media print {
+    body {
+        font-size: 12pt;
+    }
+}
+
+
+@media only screen and (max-width: 80px){
+    body{
+        background-color: gray;
+    }
+
+}
+
+
+@media only screen and (max-width: 800px){
+    #nav-bar{
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+  .product{
+    border: solid #f000f0 3px;
+    border-radius: 2px;
+    text-align: center;
+  }
+    
+}
+    </style>
+  </head>
+
+  <body>
+
+    <header id="header">
+
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png">
+
+      <nav id="nav-bar">
+        <ul>
+          <a class="nav-link" href="#features">Features</a>
+          <a class="nav-link" href="#howitworks">How It works</a>
+          <a class="nav-link" href="#pricing">Pricing</a>
+        </ul>
+      </nav>
+
+    </header>
+
+    <div>
+      <h1>Handcrafted, home-made masterpieces</h1>
+
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="GET">
+        <input type="email" name="email" id="email" placeholder="Enter your email address">
+        <button type="submit" id="submit">Submit</button>
+      </form>
+
+    </div>
+
+
+    <section id="features">
+      <div>
+        <h2>Premium Materials</h2>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+        <h2>Fast Shipping</h2>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        <h2>Quality Assurance</h2>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+    </section>
+
+    <section id="howitworks">
+
```

Comment:
here I can see that you opened a div right below a section and closed that section before closing the div. since the section act the same as the div I think this div here was unnecessary. you can use only the section.